src/nvim/globals.h

```diff
@@ -476,10 +476,6 @@ EXTERN buf_T *curbuf INIT( = NULL);    // currently active buffer
 #define FOR_ALL_BUF_WININFO(buf, wip) \
   for ((wip) = (buf)->b_wininfo; (wip) != NULL; (wip) = (wip)->wi_next)   // NOLINT
 
-// Iterate through all the signs placed in a buffer
-#define FOR_ALL_SIGNS_IN_BUF(buf, sign) \
```

Comment:
I wonder if an elaborate `FOR_ALL_MARKS` macro or function with function pointer argument would be useful for reducing boilerplate. Or would it only obfuscate unnecessarily?

Reply:
Shouldn't be a need, direct control of the iterator is better and is not much boiler plate in the simple case.